specs/tables.md

```diff
@@ -202,6 +202,20 @@ From this table, the following columns contain values using the RLC encoding:
 - ValuePrev
 - ValueCur
 
+## Keccak Table
+
+See [tx.py](src/zkevm_specs/tx.py)
+
+| IsEnabled | InputRLC   | InputLen | Output      |
+| --------- | ---------- | -------- | ----------- |
+| 1         | $input_rlc | 200      | $output_rlc |
+
+Column names in circuit:
+- IsEnabled: `is_enabled`
```

Comment:
Does the `is_enabled` column map to the `state_tag` column in the keccak circuit directly?  If so, does that mean that the tag value `Finalize` is `1`?

Reply:
The recently merged keccak has an `is_final` column, so it's a binary approach. https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/d16ac9b8bda39c45c82a41d2d59054c85abbc9a1/zkevm-circuits/src/keccak_circuit/keccak_packed_multi.rs#L824

I'm thinking of nuking the current keccak spec and then adding a new spec to reflect the new implementation.

Reply:
> The recently merged keccak has an `is_final` column, so it's a binary approach. https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/d16ac9b8bda39c45c82a41d2d59054c85abbc9a1/zkevm-circuits/src/keccak_circuit/keccak_packed_multi.rs#L824

Thanks for the clarification!

> I'm thinking of nuking the current keccak spec and then adding a new spec to reflect the new implementation.

Sounds good!